standards.md

```diff
@@ -0,0 +1,315 @@
+
+One lightweight method of satisfying the above goals is simply through HTML:
+
+The query of observational datasets could be performed by downloading a JSON formatted index file from the data server. The JSON file would contain URLs of each observational dataset and their corresponding version along with checksums to ensure the data is uncorrupted on download.  
+
+Downloading of a particular observational dataset would then simply require downloading of all files linked in the JSON file and placing them in the observational data repository on the local system.
+
+## Common Metric Output Format <a name="commonmetric"></a>
+
+Here we describe the common format for metric information calculated by DEMs. The output file format described herein is designed to allow for easy visualization and interaction within the CMEC visualization module.
+
+The expected format of the common model evaluation metric output file is JSON. This format allows for the file to be easily parsed and analyzed in the JavaScript-based visualization module. The required base-level keys are DIMENSIONS, and RESULTS. Optional keys are PROVENANCE, DISCLAIMER, and NOTES, along with any other keys required by the DEM.
+ 
+### DIMENSIONS <a name="dimensions"></a>
+
+The base-level key DIMENSIONS is required to contain at least one sub-key json_structure which is a list of dimensions used in this file for organizing metric information, arranged in a hierarchical order. The order chosen is at the discretion of the model evaluation package developer. Optional metadata can be provided to describe dimensions such as “region” or “season”.
+
+The following is an example of the DIMENSIONS key from the PCMDI Metrics Package output. The required field is DIMENSIONS:json_structure. The other metadata fields shown are optional.
+``` 
+"DIMENSIONS": {
+    "json_structure": [
+        "region",
+        "model",
+        "metric"
+    ]
+    "metric": {
+        "rms_xyt": {
+            "Name": "Spatio-Temporal Root Mean Square",
+            "Abstract": "Compute Spatial and Temporal Root Mean Square",
+            "URI": "http://uvcdat.llnl.gov/documentation/utilities/utilities-2.html"
+        },
+    },
+    "region": {
+        "Global": {},
+        "TROPICS": {
+            "generator": "cdutil.region.domain(latitude=(-30.0, 30))"
+        }
+    }
+},
+```
+Developers should consider minimizing the number of dimensions used for compatibility with the JavaScript-based visualization module. Currently, files with up to 4 dimensions are compatible with the visualization module. 
```

Comment:
Is the JavaScript visualization module a CMEC-specific element?  It might be good to clarify that is is separate from EMDS.

Reply:
This refers to a visualization module by Min Xu at ORNL. It works with any metrics data that follows the metrics JSON format set out in this doc. I'll add some more information.

Reply:
I've made the language more general to web-based tools as a concept.